t/venom/pfservers/global_config/15_create_network_devices.yml

```diff
@@ -141,3 +141,166 @@ testcases:
+        "cliAccess": null,
+        "cliEnablePwd": null,
+        "cliPwd": null,
+        "cliTransport": null,
+        "cliUser": null,
+        "coaPort": null,
+        "controllerIp": "{{.wireless01_mgmt_ip}}",
+        "deauthMethod": null,
+        "defaultAccessList": null,
+        "defaultRole": null,
+        "defaultUrl": null,
+        "defaultVlan": null,
+        "description": "{{.wireless_switch_group.description}}",
+        "disconnectPort": null,
+        "gamingAccessList": null,
+        "gamingRole": null,
+        "gamingUrl": null,
+        "gamingVlan": null,
+        "guestAccessList": null,
+        "guestRole": null,
+        "guestUrl": null,
+        "guestVlan": null,
+        "id": "{{.wireless_switch_group.id}}",
+        "inlineAccessList": null,
+        "inlineRole": null,
+        "inlineTrigger": null,
+        "inlineUrl": null,
+        "inlineVlan": null,
+        "isolationAccessList": null,
+        "isolationRole": null,
+        "isolationUrl": null,
+        "isolationVlan": null,
+        "macSearchesMaxNb": null,
+        "macSearchesSleepInterval": null,
+        "mac_trigger": null,
+        "{{.wireless_dot1x_eap_peap.roles.ad_user.id}}AccessList": null,
+        "{{.wireless_dot1x_eap_peap.roles.ad_user.id}}Role": null,
+        "{{.wireless_dot1x_eap_peap.roles.ad_user.id}}Url": null,
+        "{{.wireless_dot1x_eap_peap.roles.ad_user.id}}Vlan": "{{.wireless_dot1x_eap_peap.roles.ad_user.vlan_id}}",
+        "mode": "production",
+        "port_trigger": null,
+        "radiusSecret": "{{.wireless_switch_group.radiusSecret}}",
+        "registrationAccessList": null,
+        "registrationRole": null,
+        "registrationUrl": null,
+        "registrationVlan": null,
+        "ssid_trigger": null,
+        "type": "Hostapd_template",
+        "uplink": null,
+        "uplink_dynamic": null,
+        "useCoA": "Y",
+        "{{.wireless_dot1x_eap_peap.roles.ad_machine.id}}AccessList": null,
+        "{{.wireless_dot1x_eap_peap.roles.ad_machine.id}}Role": null,
+        "{{.wireless_dot1x_eap_peap.roles.ad_machine.id}}Url": null,
+        "{{.wireless_dot1x_eap_peap.roles.ad_machine.id}}Vlan": "{{.wireless_dot1x_eap_peap.roles.ad_machine.vlan_id}}",
+        "{{.wireless_mac_auth.roles.headless_device.id}}AccessList": null,
+        "{{.wireless_mac_auth.roles.headless_device.id}}Role": null,
+        "{{.wireless_mac_auth.roles.headless_device.id}}Url": null,
+        "{{.wireless_mac_auth.roles.headless_device.id}}Vlan": "{{.wireless_mac_auth.roles.headless_device.vlan_id}}",
+        "voiceAccessList": null,
+        "voiceRole": null,
+        "voiceUrl": null,
+        "voiceVlan": null,
+        "wsPwd": null,
+        "wsTransport": null,
+        "wsUser": null
+      }
+    headers:
+      "Authorization": "{{.pfserver_token}}"
+      "Content-Type": "application/json"
+    assertions:
+      - result.statuscode ShouldEqual 201
+
+# workaround to make accounting working:
```

Comment:
This workaround is not necessary for wireless01 because NAS-IP-Address equals wireless01 management IP address in Accounting messages.

Currently, wireless01 is only created by MAC address like an AP with controllerIP defined. 

We can keep things like this but remove that comment.